docs/tutorials/rest_api/index.md

```diff
@@ -0,0 +1,412 @@
+# Feldera: REST API
+
+:::caution Experimental feature
```

Comment:
Don't say "Experimental feature" here. I'd say it's a stable feature, just not with guaranteed backwards compatibility yet (because we're not yet v1).

```
The REST API is still evolving and might see backwards incompatible changes.
```